biometricks/src/main/java/com/handstandsam/biometricks/BiometricPromptInfo.kt

```diff
@@ -0,0 +1,27 @@
+package com.handstandsam.biometricks
+
+import androidx.biometric.BiometricPrompt
+
+/**
+ * This will force you to pass [BiometricPrompt.CryptoObject] and deviceCredentialAllowed to false.
+ *
+ * See Javadocs for [BiometricPrompt.PromptInfo.Builder]
+ *
+ * https://developer.android.com/reference/androidx/biometric/BiometricPrompt.PromptInfo.Builder
+ */
+class BiometricPromptInfo(
+    /** Required */
+    val title: String,
+    val negativeButtonText: String,
+    val cryptoObject: BiometricPrompt.CryptoObject,
+
+    /** Optional */
+    val subtitle: String? = null,
+    val description: String? = null,
+    val confirmationRequired: Boolean = true
+) {
+    /**
+     * We are going to forcing use of [BiometricPrompt.CryptoObject], so this MUST be false.
+     */
+    val deviceCredentialAllowed: Boolean = false
```

Comment:
Actually don't know much about the implications of settings this to true. I thought it would allow fallback to enter your device password and still allow decrypting the CryptoObject?

Reply:
Fair enough, I'm not 100% sure, but thought so.  Will just make it be defaulting to `false`.  Thanks.